src/highlevel_api.jl

```diff
@@ -16,35 +16,35 @@ end
 
 Prints the topology of the given `obj` as a tree to `io`.
 """
-function print_topology(io::IO = stdout, obj::Object = gettopology(); indent = "", newline = false, prefix = "")
+function print_topology(io::IO=stdout, obj::Object=gettopology(); indent="", newline=false, prefix="")
```

Comment:
This PR has a lot of formatting changes.

Could you open a separate PR that only applies your formatting changes? (I assume you used an automatic tool for this.) This could be quickly merged, and then your real changes are better visible here.

Reply:
Argh, VS Code did this by itself. Will try to undo.

Reply:
Done.